### ISSUE TITLE
required make upload fail

### ISSUE THREAD

**Issue by aemaddin** [dcasia/nova-filepond#2]
Steps to reproduce the error:-

1- setup new Laravel project
2- setup fresh nova
3- add posts table migration

```
public function up()
    {
        Schema::create('posts', function (Blueprint $table) {
            $table->bigIncrements('id');
            $table->string('image');
            $table->string('title');
            $table->text('description');
            $table->timestamps();
        });
    }
```

4- add Post model with
```
protected $fillable = [
        'title', 'description', 'image',
    ];
```

5- add nova Post resource

```
public function fields(Request $request)
    {
        return [

            ID::make()->sortable(),
            Filepond::make("image")
                ->disk("public", "posts")
                ->withoutDoka(),

            Text::make("title")
                ->rules("required"),

            Trix::make("description"),
        ];
    }
```

go to nova and try to upload a new image
<img width="1160" alt="Screen Shot 2019-10-05 at 11 00 56 PM" src="https://user-images.githubusercontent.com/11630742/66260840-4d6b6180-e7c4-11e9-8654-165330d5f829.png">

**Comment by milewski** [comment]
@aemaddin can you check your console / network logs to see what's the error you are getting?

**Comment by milewski** [comment]
Fixed on version [v0.2.2](https://github.com/dcasia/nova-filepond/releases/tag/v0.2.2),

and by the way, you don't need to add `->withoutDoka()` if you havent configured it [#enabling-doka-image-editor](https://github.com/dcasia/nova-filepond#enabling-doka-image-editor)

**Comment by aemaddin** [comment]
👍 

Thanks
